galaxy_ng/app/common/openapi.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+log = logging.getLogger(__name__)
+log.debug('openapi autoschema')
+
+
+def preprocess_debug_logger(endpoints, **kwargs):
+    """Return an iterable of (path, path_regex, method, callback) tuples"""
+
+    log.debug("kwargs: %s", repr(kwargs))
+
+    return [
+        (path, path_regex, method, callback) for path, path_regex, method, callback in endpoints
+        if not log.debug('path=%s, path_regex=%s, methos=%s, callback=%s',
+                         path, path_regex, method, callback)
```

Comment:
curious what `preprocess_debug_logger()` is used for... and does `log.debug(...)` sometimes evaluate to a boolean false by not returning any result?

Reply:
nah, log.debug() always returns nothing. That is leftover from some c&p from the other preprocess function

Reply:
I take it back, forgot it was in the 'if' clause of the generator, and the if `not log.debug()` ends up if True, so all the elements are returned.

Reply:
I've refactored that a bit to look less weird. I'll update after a './compose build' cycle to catch up some deps to test.